File: 0203-remove-linked-list-elements/0203-remove-linked-list-elements.java
class Solution {
    public ListNode removeElements(ListNode head, int val) {
        ListNode result = new ListNode(-1, head);
        ListNode now = result;

        while (now.next != null) {
            if (now.next.val == val) {
                now.next = now.next.next;
            } else {
                now = now.next;
            }
        }

        return result.next;
    }
}

File: 0203-remove-linked-list-elements/0203-remove-linked-list-elements.py
class Solution:
    def removeElements(self, head: Optional[ListNode], val: int) -> Optional[ListNode]:
        result = ListNode(-1)
        result.next = head

        now = result

        while now.next:
            if now.next.val == val:
                now.next = now.next.next
            else:
                now = now.next

        return result.next